[BUG] Error purging traceroutes. Please try again. (and similar)

**Describe the bug**
Can't execute 'Purge Telemetry', 'Reset Traceroutes' and similar.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'Settings -> Danger Zone'
2. Click on 'Purge Telemetry' or 'Reset Traceroutes'
4. See error:
popup 'Error purging traceroutes. Please try again.' or similar
http 500 on /api/purge/traceroutes
an error in the `docker logs meshmonitor --tail 50`:
```
[ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
    at Proxy.<anonymous> (file:///app/dist/services/database.js:243:35)
    at DatabaseService.purgeAllTraceroutes (file:///app/dist/services/database.js:6524:17)
    at file:///app/dist/server/server.js:4690:25
    at Layer.handleRequest (/app/node_modules/router/lib/layer.js:152:17)
    at next (/app/node_modules/router/lib/route.js:157:13)
    at file:///app/dist/server/auth/authMiddleware.js:180:13
    at process.processTicksAndRejections (node:internal/process/task_queues:104:5)
```

**Expected behavior**
Actually Purge Telemetry or Reset Traceroutes or whatever successfully.

**Environment (please complete the following information):**
* Host OS: Ubuntu 22.04.5 LTS jammy on Orange Pi Zero 3
* Deployment type: 3.8.7 using Docker:
```
docker inspect meshmonitor | grep Image
        "Image": "sha256:d5315d693d5b8955421c18fe7ea9f6a9cb6b6cdc36221e8c6b5d617d2ec1e39f",
            "Image": "ghcr.io/yeraze/meshmonitor:latest",
```
on `Docker version 29.3.0, build 5927d80`
* See no networking-related issues.
* Database backend: `PostgreSQL 18.3 (Ubuntu 18.3-1.pgdg22.04+1) on aarch64-unknown-linux-gnu, compiled by gcc (Ubuntu 11.4.0-1ubuntu1~22.04.3) 11.4.0, 64-bit`
* Node type: Heltec V4, but don't think it's related
* Node Connection Type: Wifi
* Not about VirtualNode

**Additional context**
I use my own external PostgreSQL, defined as 
```
...
    environment:
    ...
      - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@192.168.88.5:54321/meshmonitor
```
in the compose file

also
```
docker compose logs meshmonitor | grep -i "Drizzle"
meshmonitor  | [INFO] [DatabaseService] Using PostgreSQL driver for Drizzle repositories
meshmonitor  | [INFO] [DatabaseService] Drizzle repositories initialized successfully
meshmonitor  | [INFO] [DatabaseService] Using PostgreSQL driver for Drizzle repositories
meshmonitor  | [INFO] [DatabaseService] Drizzle repositories initialized successfully
meshmonitor  | [INFO] [DatabaseService] Using PostgreSQL driver for Drizzle repositories
meshmonitor  | [INFO] [DatabaseService] Drizzle repositories initialized successfully
meshmonitor  | [ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
meshmonitor  | [ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
meshmonitor  | [ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
meshmonitor  | [ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
meshmonitor  | [ERROR] Error purging telemetry: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
meshmonitor  | [ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
meshmonitor  | [ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
meshmonitor  | [ERROR] Error purging traceroutes: Error: SQLite method 'exec' called but using postgres database. Use Drizzle repositories instead.
```
I don't get it, isn't meshmonitor already using Drizzle?
or not all the code was upgraded to PostgreSQL compat, i guess?

hope I gave all the necessary info. feel free to ask any extra info, ofc.

--- a/src/services/database.ts
+++ b/src/services/database.ts
@@ -7745,9 +7745,13 @@ class DatabaseService {
     // For PostgreSQL/MySQL, use async repository
     if (this.drizzleDbType === 'postgres' || this.drizzleDbType === 'mysql') {
       if (this.telemetryRepo) {
-        this.telemetryRepo.deleteAllTelemetry().catch(err => {
+        this.telemetryRepo.deleteAllTelemetry().then(() => {
+          logger.debug('✅ Successfully purged all telemetry');
+        }).catch(err => {
           logger.error('Failed to purge all telemetry:', err);
         });
+      } else {
+        logger.warn('Cannot purge telemetry: telemetry repository not initialized');
       }
       return;
     }
@@ -7871,9 +7875,12 @@ class DatabaseService {
         this.messagesRepo.deleteAllMessages().then(() => {
           // Clear messages cache after purge
           this._messagesCache = [];
+          logger.debug('✅ Successfully purged all messages');
         }).catch(err => {
           logger.error('Failed to purge all messages:', err);
         });
+      } else {
+        logger.warn('Cannot purge messages: messages repository not initialized');
       }
       return;
     }
@@ -7895,6 +7902,8 @@ class DatabaseService {
         }).catch(err => {
           logger.error('Failed to purge all traceroutes:', err);
         });
+      } else {
+        logger.warn('Cannot purge traceroutes: traceroutes repository not initialized');
       }
       return;
     }

